shadows/framework/src/main/java/org/robolectric/shadows/ShadowContentResolver.java

```diff
@@ -530,6 +531,26 @@ protected static boolean isSyncActive(Account account, String authority) {
     return status != null && status.syncRequests > 0;
   }
 
+  @Implementation
+  protected static List<SyncInfo> getCurrentSyncs() {
+    List<SyncInfo> list = new ArrayList<>();
+    if (syncableAccounts == null) {
+      return null;
```

Comment:
Oh, I remember the result of this method can't be null. Could you check official documentation of this method?

Reply:
Yeah, syncableAccounts==null is always going to false.
I think we should remove it from the code.
It wouldn't impact the rest of code.

Reply:
I removed it and now it's working fine.
Can i finally commit again?